api/turing/api/router_versions_api.go

```diff
@@ -177,6 +180,24 @@ func (c RouterVersionsController) DeployRouterVersion(
 		"version":   int(routerVersion.Version),
 	})
 }
+func (c RouterVersionsController) ListRouterVersionsByEnsembler(
```

Comment:
Since we're changing the name of the API endpoint, can we also change the name of the controller method? :) Thanks!